petclinic-frontend/src/features/customers/components/CustomerDetails.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import './CustomerDetails.css';
+
+const CustomerDetails: FC = () => {
+  const { ownerId } = useParams<{ ownerId: string }>();
+  const navigate = useNavigate();
+
+  const handleEditClick = (): void => {
+    navigate(`/customers/${ownerId}/edit`);
+  };
+
+  return (
+    <div className="customer-details-card">
+      <h2>Customer Details</h2>
+      <p>
+        <strong>First Name: </strong>
+      </p>
+      <p>
+        <strong>Last Name: </strong>
+      </p>
+      <p>
+        <strong>Address: </strong>
+      </p>
+      <p>
+        <strong>City: </strong>
+      </p>
+      <p>
+        <strong>Province: </strong>
+      </p>
```

Comment:
You can change the Province field to be a drop-down menu for easier data entry.
```suggestion
      <p>
        <strong>Province: </strong>
        <select value={province} onChange={handleProvinceChange}>
          <option value="" disabled>Select Province/Territory</option>
          <option value="Alberta">Alberta</option>
          <option value="British Columbia">British Columbia</option>
          <option value="Manitoba">Manitoba</option>
          <option value="New Brunswick">New Brunswick</option>
          <option value="Newfoundland and Labrador">Newfoundland and Labrador</option>
          <option value="Nova Scotia">Nova Scotia</option>
          <option value="Ontario">Ontario</option>
          <option value="Prince Edward Island">Prince Edward Island</option>
          <option value="Quebec">Quebec</option>
          <option value="Saskatchewan">Saskatchewan</option>
          <option value="Northwest Territories">Northwest Territories</option>
          <option value="Nunavut">Nunavut</option>
          <option value="Yukon">Yukon</option>
        </select>
      </p>
```

Reply:
I think that's a great suggestion since there aren't many provinces in Canada.